.release.cloudbuild.yaml

```diff
@@ -201,6 +201,16 @@ steps:
   id:   'copyPythonComponentSDKToLatest'
   waitFor: ['copyPythonComponentSDKLocal']
 
+# Generate and copy the pipeline-lite deployment YAML
+- name: 'google/cloud-sdk'
```

Comment:
Nit: Tasks are mush easier to parse when they're separated by an empty line and the `id` is the first attribute.